### PR TITLE
Fix: UnitResult gets nplural data and passes that pluralize_target util

### PR DESCRIPTION
**Pull request by phlax**
prevents pluralize_target from trying to fetch nplural data for a UnitResult

### CODE REVIEW

File: pootle/apps/pootle_store/unit/results.py [hunk 1]
```diff
@@ -22,6 +22,11 @@
 class UnitResult(UnitProxy):
 
     @property
+    def nplurals(self):
+        return self.unit[
+            "store__translation_project__language__nplurals"] or 0
```
Comment (unho) [inline pootle/apps/pootle_store/unit/results.py]: This default surely is not the best choice. Perhaps it is better to use 1 or 2.

Comment (phlax) [inline pootle/apps/pootle_store/unit/results.py]: i think the default is correct - im just trying prevent ``None`` from being passed

Comment (phlax) [inline pootle/apps/pootle_store/unit/results.py]: ie - this is the default on the model

Comment (unho) [inline pootle/apps/pootle_store/unit/results.py]: You are right. Sorry.

Comment (julen) [inline pootle/apps/pootle_store/unit/results.py]: I think this should fallback to `1`.

Comment (julen) [inline pootle/apps/pootle_store/unit/results.py]: Oh, didn't get your messages up to the point. If it's tied to the model, then `0` is good.

**Comment by julen** [comment]
Tested and works fine, thank you for the quick fix!

File: pootle/apps/pootle_store/unit/results.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_store/unit/results.py]: @julen im not really sure what it should be but im reluctant to make it different to the default on the model - https://github.com/translate/pootle/blob/master/pootle/apps/pootle_language/models.py#L92

**Pull request merged**